README.md

```diff
@@ -25,7 +25,27 @@ TRELLO_DONE_LIST_ID
 ```bash
 vagrant up
 ```
+## Running within Docker
 
+### Building docker image
+To build the docker image run the following command
```

Comment:
Really minor, but this isn't quite what you mean! `Run the relevant command for your target environment` might be clearer, or just split them out